corehq/apps/users/models.py

```diff
@@ -1992,9 +1992,12 @@ def __repr__(self):
             self=self
         ))
 
+    def get_usercase(self):
+        return CaseAccessors(self.domain).get_case_by_domain_hq_user_id(self._id, USERCASE_TYPE)
+
     @skippable_quickcache(['self._id'], lambda _: settings.UNIT_TESTING)
     def get_usercase_id(self):
-        case = CaseAccessors(self.domain).get_case_by_domain_hq_user_id(self._id, USERCASE_TYPE)
+        case = self.get_usercase()
```

Comment:
@snopoke not sure i understand - i didn't want to bypass the cache, i just wanted to get the actual user case and not just the user case id